.circleci/config.yml

```diff
@@ -806,7 +806,7 @@ jobs:
                 -X POST \
                 -H 'Content-Type: application/json' \
                 -H 'Accept: application/json' \
-                -d "{\"branch\": \"master\",\"parameters\":{\"SOURCE_REPO\": \"${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}\",\"SOURCE_TAG\": \"${CIRCLE_TAG}\"}}" \
+                -d "{\"branch\": \"master\",\"parameters\":{\"SOURCE_REPO\": \"${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}\",\"SOURCE_TAG\": \"0.33.0\",\"TEMPLATE_DIR\": \"charts/consul\",\"SOURCE_BRANCH_OVERRIDE\": \"master\"}}" \
```

Comment:
This line should be like below where you replace 0.33.0 with the version you are actually releasing
```
-d "{\"branch\": \"master\",\"parameters\":{\"SOURCE_REPO\": \"${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}\",\"SOURCE_TAG\": \"0.33.0\",\"TEMPLATE_DIR\": \"charts/consul\"}}" \
```